src/corner_kick/scripts/start_test.sh

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
 # Start test script for Corner Kick.
 
+export PERCY_TOKEN=369660795fa79afd24f25e21a4f8803e9d56beae94a31fbd4024ad34cf1c8cd1
```

Comment:
In general I think private tokens like this shouldn't be committed into code, but rather should be stored in travis ci (so they're not public). I'd can point you to where it should go (or just move it over myself) if that sounds reasonable?

Reply:
Sadly, Travis CI does not give us access to secrets from forked branch PRs

> If you fork a project and add it to Travis CI, it will not have access to the encrypted variables.

Reply:
Hmm, what does having this token allow you to do?

Reply:
Removed Percy from scope